modules/react-map-gl-draw/src/editor.js

```diff
@@ -138,6 +144,36 @@ export default class Editor extends PureComponent<EditorProps, EditorState> {
   _events: any;
   _context: ?MapContext;
 
+  _onKeydown = (evt: MjolnirEvent) => {
+    const { features } = this.state;
+    if (
+      features &&
+      (evt.key === 'Backspace' || evt.key === 'Delete') &&
```

Comment:
i would recommend to user event.which https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/which in order to detect the key.
backspace: 8
delete: 46

I would create constants for those values